src/Microsoft.Health.Fhir.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -18,6 +18,6 @@ public static class SchemaVersionConstants
         public const int SupportsResourceChangeCaptureSchemaVersion = (int)SchemaVersion.V14;
         public const int BulkReindexReturnsFailuresVersion = (int)SchemaVersion.V16;
         public const int AddMinMaxForDateAndStringSearchParamVersion = (int)SchemaVersion.V18;
-        public const int SupportsPartitionedResourceChangeDataVersion = (int)SchemaVersion.V20;
+        public const int SupportsPartitionedResourceChangeDataVersion = (int)SchemaVersion.V21;
```

Comment:
I am wondering if this change would break the existing check in [SqlServerFhirResourceChangeDataStore](https://github.com/microsoft/fhir-server/blob/0d6e90c7d17dd1273147eb84fcfd24ccc5857e7b/src/Microsoft.Health.Fhir.SqlServer/Features/ChangeFeed/SqlServerFhirResourceChangeDataStore.cs#L165)?

Reply:
Thank you for pointing it out. Added new schema version constant.